src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let vocationCost = 0;
+
+  if (days >= 7) {
+    vocationCost += days * cost - bigDiscount;
+  } else if (days >= 3) {
+    vocationCost += days * cost - smallDiscount;
```

Comment:
Instead of using 'vocationCost +=', you can directly assign the value to 'vocationCost' since it is initialized to 0.